app/controllers/comments_controller.rb

```diff
@@ -18,8 +18,8 @@ def create
   end
 
   def update
-    if current_user.campaigns.include? campaign | current_admin
-      comment = Comment.find params[:id]
+    comment = Comment.find params[:id]
```

Comment:
**A small note:** This violates our convention of using parentheses around method params.

`Comment.find(params[:id])`